lib/nudge_web/controllers/event_controller.ex

```diff
@@ -0,0 +1,50 @@
+defmodule NudgeWeb.EventController do
+  use NudgeWeb, :controller
+  alias Nudge.Sites
+  alias Nudge.Sites.Event
+
+  def new(conn, _params) do
+    changeset = Sites.change_event(%Event{})
+    render(conn, "new.html", changeset: changeset)
+  end
+
+  def create(conn, %{
+        "event" => %{
+          "title" => title,
+          "body" => body,
+          "start_date" => start_date,
+          "end_date" => end_date,
+          "kind" => kind,
+          "respect_dnd" => dnd
+        }
+      }) do
+    IO.inspect(conn.assigns)
+
+    case Sites.create_event(%{
+           "title" => title,
+           "body" => body,
+           "start_date" => start_date,
+           "end_date" => end_date,
+           "kind" => kind,
+           "respect_dnd" => dnd,
+           "site_id" => 7
+         }) do
+      {:ok, event} ->
+        conn
+        |> put_flash(:info, "Site created successfully.")
+        |> redirect(to: Routes.site_event_path(conn, :index, :id))
```

Comment:
`redirect(to: Routes.site_event_path(conn, :index, :id))`

Will this line work? Shouldn't we be passing the id of the site?